index.html

```diff
@@ -1,3 +1,46 @@
-﻿
-
+<html lang ="en">
+<img src="coderbabez-hw-01-html-basics-megknoll/img/cookies.jpg" alt="Cookies">
+<<head>The Best Chocolate Chip Cookies By Grandma
+</head>
+<<body>
+  <<header>
+    <<h1>The Best Chocolate Chip Cookies</h1>
+    <<h2><<i>Recipe by Grandma</i></h2>
+    <<h3>Prep Time: 45 minutes</h3>
+  </header><
+  <section><<h3>Ingredients</h3>
+  <<p> <ul>
+  <li>1 1/2 cups (3 sticks) softened butter</li>
+  <li>1 cup brown sugar</li>
+  <li>1 cup granulated sugar</li>
+  <<li>1 Tbl vanilla instant pudding powder</li>
+  <<li>2 Tbl milk</li>
+  <<li>2 Tbl vanilla extract</li>
+  <<li>2 eggs</li>
+  <<li>4 cups all purpose flour</li>
+  <<li>2 tsp baking soda</li>
+  <<li>1/2 tsp salt</li>
+  <<li>4 cups chocolate chips</li>
+  <<li>1 cup chopped walnuts or pecans(optional)</li>
 
+</ul></p></section>
+<<section>
+  <<header>
```

Comment:
`<header>` tags are often reserved for content that would serve as the header of the site, kind of like a navigation bar. In this instance, you'd just want to use the `<h5`> element, which is more like a style notation.